taxi/tests/test_form.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm, DriverLicenseUpdateForm
+
+
+class DriverCreationFormTests(TestCase):
+    def test_driver_creation_form(self):
+        form_data = {
+            "username": "testuser",
+            "password1": "testpa10-ssworD",
+            "password2": "testpa10-ssworD",
+            "first_name": "First",
+            "last_name": "Second",
+            "license_number": "ADF12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` might fail because `cleaned_data` typically contains additional fields or transformations not present in the original `form_data`. Consider checking only the relevant fields or using `form.cleaned_data.get('field_name')` for assertions.